blocks/gu_spoverview/block_gu_spoverview.php

```diff
@@ -43,13 +52,90 @@ public function hide_header() {
      *       See myoverview plugin for reference.
      */
     public function get_content() {
-        if ($this->content !== null) {
-          return $this->content;
+        global $USER, $DB, $PAGE, $OUTPUT;
+
+
+        $PAGE->requires->css('/blocks/assessmentglance/styles.css');
+
+        $student_id = $USER->id;
+        $current_yr = date("Y");
+
+        // I took all the assignments whose status was submitted
```

Comment:
Do not use first person in comments. Sample comment should be: // Query for getting all submitted assignments